airbyte-webapp/src/pages/routes.tsx

```diff
@@ -153,6 +154,15 @@ const RoutingWithWorkspace: React.FC = () => {
 };
 
 export const Routing: React.FC = () => {
+  const { search } = useLocation();
```

Comment:
:information_source: Technically this isn't really needed at the moment, since we can't have user registration in OSS. I'd still prefer us adding this already, so we won't accidentally forget it, when we merge the cloud App and OSS App together, or in case we have users in OSS. The performance overhead is nearly non existing for this.

Reply:
This part is not required for oss, so we probably should remove it. If at some point we need to combine oss and cloud - then it could be injected properly via modules approach.